test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -251,7 +251,7 @@ public static void setRotation(@Rotation int rotation) {
      *
      * @param enabled What state the spell checker should be set to
      */
-    public static void setSpellCheckerEnabled(boolean enabled) {
+    public static void setSpellCheckerState(boolean enabled) {
```

Comment:
This is technically a breaking change, but it should be okay since we haven't released this yet :)